pkg/controllers/provisioning/binpacking/packable.go

```diff
@@ -203,52 +186,41 @@ func (p *Packable) validateOfferings(constraints *v1alpha5.Constraints) error {
 	return fmt.Errorf("offerings %v are not available for capacity types %s and zones %s", p.Offerings(), constraints.Requirements.CapacityTypes(), constraints.Requirements.Zones())
 }
 
-func (p *Packable) validateGPUs(pods []*v1.Pod) error {
-	gpuResources := map[v1.ResourceName]*resource.Quantity{
-		resources.NvidiaGPU: p.InstanceType.NvidiaGPUs(),
-		resources.AMDGPU:    p.InstanceType.AMDGPUs(),
-		resources.AWSNeuron: p.InstanceType.AWSNeurons(),
-	}
-	for resourceName, instanceTypeResourceQuantity := range gpuResources {
-		if p.requiresResource(pods, resourceName) && instanceTypeResourceQuantity.IsZero() {
-			return fmt.Errorf("%s is required", resourceName)
-		} else if !p.requiresResource(pods, resourceName) && !instanceTypeResourceQuantity.IsZero() {
```

Comment:
This is a change from previous behavior.  For the three known GPU types, it would mark instance types that had those GPUs as incompatible with pods that didn't require them.  

The change I've made leaves them as compatible since you can run a pod that doesn't require resource XYZ on an instance type that has resource XYZ.